netsim/outputs/_graph.py

```diff
@@ -44,13 +44,15 @@ def build_nodes(topology: Box, g_type: str) -> Box:
     for name,n in topology.nodes.items():
       bgp_as = n.get('bgp.as',None)
       if bgp_as:
-        bgp_as = f'AS_{bgp_as}'
-        maps.bgp[bgp_as].nodes[n.name] = n
+        map_asn = f'AS_{bgp_as}'
```

Comment:
This could be a local function, the format has to match with line 53 below

Reply:
In principle, you're right. In practice, we'll probably survive ;)